src/main/java/com/minecolonies/coremod/client/render/TileEntityInfoPosterRenderer.java

```diff
@@ -0,0 +1,212 @@
+            f2 = 180.0F;
+        }
+
+        if (k == 4)
+        {
+            f2 = 90.0F;
+        }
+
+        if (k == 5)
+        {
+            f2 = -90.0F;
+        }
+        GlStateManager.rotate(-f2, 0.0F, 1.0F, 0.0F);
+
+        for (final BlockRenderLayer layer : BlockRenderLayer.values())
+        {
+            if (ModBlocks.blockInfoPoster.canRenderInLayer(actualState, layer))
+            {
+                final EntityPlayer player = Minecraft.getMinecraft().thePlayer;
+                final double dx = player.lastTickPosX + (player.posX - player.lastTickPosX) * partialTicks;
+                final double dy = player.lastTickPosY + (player.posY - player.lastTickPosY) * partialTicks;
+                final double dz = player.lastTickPosZ + (player.posZ - player.lastTickPosZ) * partialTicks;
+
+                GlStateManager.pushMatrix();
+                GlStateManager.translate(pos.getX() - dx, pos.getY() - dy, pos.getZ() - dz);
+
+                RenderHelper.disableStandardItemLighting();
+
+                if (layer == BlockRenderLayer.CUTOUT)
+                {
+                    Minecraft.getMinecraft().getTextureManager().getTexture(SIGN_TEXTURE).setBlurMipmap(false, false);
+                }
+
+                GlStateManager.color(1F, 1F, 1F, 1F);
+
+                final int alpha = ((int) (1.0D * 0xFF)) << 24;
+
+                GlStateManager.enableBlend();
+                GlStateManager.enableTexture2D();
+
+                GlStateManager.blendFunc(GL11.GL_SRC_ALPHA, GL11.GL_ONE_MINUS_SRC_ALPHA);
+                GlStateManager.colorMask(false, false, false, false);
+                this.renderModel(world, model, pos, alpha);
+
+                GlStateManager.colorMask(true, true, true, true);
+                GlStateManager.depthFunc(GL11.GL_LEQUAL);
+                this.renderModel(world, model, pos, alpha);
+
+                GlStateManager.disableBlend();
+
+                if (layer == BlockRenderLayer.CUTOUT)
+                {
+                    Minecraft.getMinecraft().getTextureManager().getTexture(SIGN_TEXTURE).restoreLastBlurMipmap();
+                }
+
+                GlStateManager.popMatrix();
+            }
+        }
+
+
+        GlStateManager.enableRescaleNormal();
+        GlStateManager.pushMatrix();
+        final FontRenderer fontrenderer = this.getFontRenderer();
+        GlStateManager.translate(0.0F, 0.33333334F, 0.046666667F);
+        GlStateManager.scale(0.010416667F, -0.010416667F, 0.010416667F);
+        GlStateManager.glNormal3f(0.0F, 0.0F, -0.010416667F);
+        GlStateManager.depthMask(false);
+
+        if (destroyStage < 0)
+        {
+            for (int j = 0; j < te.signText.length; ++j)
+            {
+                if (te.signText[j] != null)
+                {
+                    ITextComponent itextcomponent = te.signText[j];
+                    List<ITextComponent> list = GuiUtilRenderComponents.splitText(itextcomponent, 90, fontrenderer, false, true);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 90 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)